ui/src/App.tsx

```diff
@@ -80,6 +85,20 @@ function App() {
     }
   };
 
+  function fetchedCameras(cameras: Camera[] | null) {
+    if (cameras !== null && cameras.length >0) {
+      return (
+        <>
+          <Route path="" element={ <ListActivity cameras={cameras}
+                                                 showSelectors={showListSelectors}
+                                                 timeZoneName={timeZoneName!}
+          />} />
+          <Route path="live" element={<LiveActivity cameras={cameras} layoutIndex={multiviewLayoutIndex} />} />
```

Comment:
There's no matching route on the server to make this work. I did some reading, and I think the options are:

*   we could modify the server to serve `index.html` when given this path, but we'll need to update the Rust code when we extend the UI, which might get messy.
*   we could use react-router's [`HashRouter`](https://v5.reactrouter.com/web/api/HashRouter).

I'm leaning toward the latter; what do you think?